### PR TITLE
[15.0][IMP] website_sale_product_attribute_filter_collapse: changes to the template

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
This module has an old template design, it has been modified to maintain harmony with the current odoo design.

cc @tecnativa TT41385

@chienandalu   @CarlosRoca13  please review :)

ping @pedrobaeza

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Can you add a screenshot of the before and after?

**Comment by pilarvargas-tecnativa** [comment]
Some screenshots of the old design that does not follow the same lines of the odoo design and the modification in the collapse of the attributes. 

Before:
image

After:
image

File: website_sale_product_attribute_filter_collapse/views/templates.xml [hunk 1]
```diff
@@ -12,22 +12,24 @@
...
-                    aria-hidden="true"
-                />
-                <i
-                    class="fa fa-angle-down"
-                    id="icon_attribute_expanded"
-                    aria-hidden="true"
-                />
-            </a>
+            <div>
+                    <a
```
Comment (pedrobaeza) [inline website_sale_product_attribute_filter_collapse/views/templates.xml]: Incorrect indentation in this file. Please normalize it.

Comment (pilarvargas-tecnativa) [reply website_sale_product_attribute_filter_collapse/views/templates.xml]: thanks :)

**Review by chienandalu — COMMENTED** [review]
Good job :+1: 

Testing it I noticed that the hover shows the links underlined:

image


Fix it to match the general style as well

**Review by hieulucky111 — APPROVED** [review]
Code review
LGTM

**Comment by pilarvargas-tecnativa** [comment]
> Good job +1
> 
> Testing it I noticed that the hover shows the links underlined:
> 
> image
> 
> Fix it to match the general style as well

thanks! :)

**Pull request merged**